packages/network/src/api/index.ts

```diff
@@ -98,4 +101,21 @@ export class NetworkClient extends BaseApiClient {
       return await api.put(`${PATH}/private/${id}`, { ...network, tenantNetworkId: id });
     });
   }
+
+  private toValidPrivateNetworkRequest(network: TenantNetworkCreateRequest): TenantNetworkCreateRequest {
+    if (network.configuration && network.configuration.safeTxServiceURL) {
+      let safeTxServiceURL = network.configuration.safeTxServiceURL;
+      if (safeTxServiceURL.endsWith('/')) {
+        safeTxServiceURL = safeTxServiceURL.slice(0, -1);
+      }
+      if (safeTxServiceURL.startsWith('https://')) {
+        safeTxServiceURL = safeTxServiceURL.replace('https://', '');
+      }
+      if (safeTxServiceURL.startsWith('http://')) {
+        safeTxServiceURL = safeTxServiceURL.replace('http://', '');
+      }
+      network.configuration.safeTxServiceURL = safeTxServiceURL;
+    }
+    return network;
+  }
```

Comment:
Nit: I wonder if it makes sense to move this logic to our backend instead of validating it here.